ckan/tests/logic/action/test_get.py

```diff
@@ -41,6 +41,15 @@ def foo(key, data, errors, context):
 
         eq(dataset2['new_field'], 'foo')
 
+
+class TestGroupList(object):
+
+    def setup(self):
+        helpers.reset_db()
+
+        # Clear the search index
+        search.clear()
```

Comment:
I think we have a FunctionalTestBase or similar that does the reset_db etc. (so you can save a few extra lines on all these classes)